### ISSUE TITLE
@nebula.js/enigma-mocker is not in NPM

### ISSUE THREAD

**Issue by jansivans** [qlik-oss/nebula.js#744]
## 🐛 Bug report

Can't install @nebula.js/enigma-mocker.

### Steps to Reproduce

1. Open terminal.
2. Type `npm install @nebula.js/enigma-mocker`.

### Expected behavior

Enigma-mocker should be installed.

### Actual behavior

Enigma-mocker is not installed.

**Comment by Caele** [comment]
Yes it is internal to Nebula at the moment, but feel free to use it as such. We will revisit publishing it as we use it more.
